### ISSUE TITLE
Scroll all in one app like Venom.

### ISSUE THREAD

**Issue by pristjones** [scroll-tech/contribute-to-scroll#270]
Scroll devs can develop an all in one app like Venom. This will help newbies coming into blockchain technology to onboard easily.

**Comment by Cephas0205** [comment]
Exactly. This would make it very user friendly and attract more people.
